packages/frontend/src/components/Explorer/ExplorerHeader/index.tsx

```diff
@@ -151,10 +153,18 @@ const ExplorerHeader: FC = memo(() => {
                 {!savedChart && userCanCreateCharts && (
                     <SaveChartButton isExplorer />
                 )}
-                <ShareShortLinkButton
-                    disabled={!isValidQuery}
-                    url={urlToShare}
-                />
+                <Can
+                    I="manage"
```

Comment:
is this manage available for all users ? I only see this used. Some functions, like  this one (sharing) could be used by editors.